docs/kb/rules/ruleset-default-mode.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Adjust the default mode for a whole ruleset.
+tags:
+  - Rules
+  - Semgrep Registry
+  - Semgrep Code
+append_help_link: true
+---
+
+# Why do new rules keep appearing in Comment or Block mode?
+
+Semgrep AppSec Platform [Policies](/docs/semgrep-code/policies) can contain both individual rules and **rulesets**, which are curated groups of rules recommended for particular purposes. All organizations in the AppSec Platform start with two rulesets: the `default` ruleset and the `comment` ruleset. These rulesets are curated to be good starter packs for security teams to review, and for developers to see comments about, respectively.
```

Comment:
```suggestion
Semgrep AppSec Platform [Policies](/docs/semgrep-code/policies) can contain both individual rules and **rulesets**, which are curated groups of rules recommended for particular purposes. All organizations start with two rulesets: the `default` ruleset, which is a good starter pack for security teams, and the `comment` ruleset, which is a good starter pack for developers.
```

I was a bit confused about what "These rulesets are curated to be good starter packs for security teams to review, and for developers to see comments about, respectively" meant  -- do you mean the `default` ruleset is good for security teams to start reviewing findings?

Reply:
yeah, your version is better - that was some bad writing on my part, sorry!